src/p2p/Join/index.ts

```diff
@@ -199,15 +204,13 @@ export function calculateToAccept(): number {
 }
 
 export function getTxs(): P2P.JoinTypes.Txs {
-  // Omar - maybe we don't have to make a copy
-  // [IMPORTANT] Must return a copy to avoid mutation
-  const requestsCopy = deepmerge({}, requests)
-  //const keepInStandbyCopy = [...Object.values(keepInStandbyCollector)]
   return {
-    join: requestsCopy,
-    startedSyncing: [],
-    finishedSyncing: [],
-    standbyRefresh: [],
+    standbyAdd: drainNewJoinRequests(),
+    startedSyncing: drainSyncStarted(),
+    finishedSyncing: drainFinishedSyncingRequest(),
+    standbyRefresh: drainNewStandbyRefreshRequests(),
+    standbyRemove: drainNewUnjoinRequests(),
+    lostAfterSelection: drainLostAfterSelectionNodes(),
```

Comment:
we use cycle txs for changes that we use gossip for. lostAfterSelection is calculated, so we shouldn't include it here. better to just use the old way we did it

Reply:
I think i explained this a bit more in the PR for the lib types change